src/body/params.ts

```diff
@@ -7,7 +7,7 @@ export class Params extends URLSearchParams {
   /**
    * Convert parameters into a plain object, useful for validation.
    */
-  toObject(): Record<string, string> {
+  toObject(): Record<string, any> {
```

Comment:
I don't think this does anything.

Reply:
it does when in TypeScript you try to take something different than a string from an inputData key.

Here is a small example of  a project I'm experimenting with:

(please correct the first import to your working mojojs path)

If you reverse the line (10 above), tsc will complain that inputData.lat and inputData.long are not numbers

```typescript
import mojo from '../../../mojo.js/lib/core.js';
import { find } from 'geo-tz';
const app = mojo();

app.validator.addSchema({
  type: "object",
  properties: {
    lat: { type: "number", minimum: -90, maximum: 90 },
    long: { type: "number", minimum: -180, maximum: 180 }
  },
  required: ["lat", "long"]
}, "geoInput");

app.get('/tz', { ext: ['json'] }).to({ ext: 'json' }, async ctx => {
  const params = await ctx.params();
  const validate = ctx.schema('geoInput');
  const inputData = params.toObject();
  const result = validate?.(inputData);
  if (result?.isValid) {
    await ctx.render({
      json: { tz: find(inputData.lat, inputData.long), all: inputData }
    });
  } else {
    await ctx.render({
      json: {
        errors: JSON.parse(JSON.stringify(result?.errors)),
        inputs: inputData
      }, status: 400
    });
  }
});

app.start();
```

Reply:
You do make a good case against type coercion, maybe we shouldn't support it. 🤔